pkgs/by-name/co/coulr/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  wrapGAppsHook4,
+  pkg-config,
+  meson,
+  ninja,
+  appstream-glib,
+  desktop-file-utils,
+  gobject-introspection,
+  libadwaita,
+  libportal-gtk4,
+  libnotify,
+  nix-update-script,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "coulr";
+  version = "2.1.0";
+
+  format = "other";
```

```suggestion
  pyproject = false;
```

(Just preferred new style)